packages/cli/internal/pkg/aws/cfn/stack_filters.go

```diff
@@ -2,25 +2,42 @@ package cfn
-	types.StackStatusRollbackComplete,
-	types.StackStatusDeleteInProgress,
-	types.StackStatusDeleteFailed,
-	types.StackStatusUpdateInProgress,
-	types.StackStatusUpdateCompleteCleanupInProgress,
-	types.StackStatusUpdateComplete,
-	types.StackStatusUpdateRollbackInProgress,
-	types.StackStatusUpdateRollbackFailed,
-	types.StackStatusUpdateRollbackCompleteCleanupInProgress,
-	types.StackStatusUpdateRollbackComplete,
-	types.StackStatusReviewInProgress,
-	types.StackStatusImportInProgress,
-	types.StackStatusImportComplete,
-	types.StackStatusImportRollbackInProgress,
-	types.StackStatusImportRollbackFailed,
-	types.StackStatusImportRollbackComplete,
+type StackOptions struct {
+	activeStack    bool
+	queryableStack bool
+}
+
+var stackDefinitions = map[types.StackStatus]StackOptions{
+	types.StackStatusCreateInProgress:                        {activeStack: true, queryableStack: false},
+	types.StackStatusCreateComplete:                          {activeStack: true, queryableStack: true},
+	types.StackStatusRollbackInProgress:                      {activeStack: true, queryableStack: false},
+	types.StackStatusRollbackFailed:                          {activeStack: true, queryableStack: false},
+	types.StackStatusRollbackComplete:                        {activeStack: true, queryableStack: false},
+	types.StackStatusDeleteInProgress:                        {activeStack: true, queryableStack: false},
+	types.StackStatusDeleteFailed:                            {activeStack: true, queryableStack: false},
+	types.StackStatusUpdateInProgress:                        {activeStack: true, queryableStack: true},
+	types.StackStatusUpdateCompleteCleanupInProgress:         {activeStack: true, queryableStack: true},
+	types.StackStatusUpdateComplete:                          {activeStack: true, queryableStack: true},
+	types.StackStatusUpdateRollbackInProgress:                {activeStack: true, queryableStack: true},
+	types.StackStatusUpdateRollbackFailed:                    {activeStack: true, queryableStack: true},
+	types.StackStatusUpdateRollbackCompleteCleanupInProgress: {activeStack: true, queryableStack: true},
+	types.StackStatusUpdateRollbackComplete:                  {activeStack: true, queryableStack: false},
+	types.StackStatusReviewInProgress:                        {activeStack: true, queryableStack: true},
+	types.StackStatusImportInProgress:                        {activeStack: true, queryableStack: true},
+	types.StackStatusImportComplete:                          {activeStack: true, queryableStack: true},
+	types.StackStatusImportRollbackInProgress:                {activeStack: true, queryableStack: true},
+	types.StackStatusImportRollbackFailed:                    {activeStack: true, queryableStack: true},
+	types.StackStatusImportRollbackComplete:                  {activeStack: true, queryableStack: true},
+}
+var ActiveStacksFilter []types.StackStatus
+var QueryableStacksMap map[types.StackStatus]bool
+
+func init() {
```

Comment:
I'm not seeing anything ever calling init here. Is there some Go magic that calls it when the package is used?

Reply:
It's some go magic :)
Docs: https://tutorialedge.net/golang/the-go-init-function/

Reply:
`Init()` is a special function that is called one upon loading that module.